### PR TITLE
feat: [BB-244] list component for fertilizer

### PR DESCRIPTION
**Pull request by kcaparas1630**
# Description

Created list component for fertilizers to show data that has been entered in the input fields.

Mockup Reference:

Group 163
Group 161

## Type of changes

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

## Further comments

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-sustainment-capstone-2024-272-frontend.apps.silver.devops.gov.bc.ca)
- [Backend](https://nr-sustainment-capstone-2024-272-backend.apps.silver.devops.gov.bc.ca/api/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-sustainment-capstone-2024/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-sustainment-capstone-2024/actions/workflows/merge.yml)

### CODE REVIEW

**Comment by kcaparas1630** [comment]
Very straightforward list, a repeat of fields list.

File: frontend/src/Commons/Forms/InputModules/Fertilizers/Fertilizers.tsx [hunk 1]
```diff
@@ -111,100 +114,204 @@ const FertilizersInfo: React.FC<InputModuleProps> = ({
...
+  //                   text={ComponentText.SAVE_FERTILIZER}
+  //                 />
+  //               </StyledButtonContainer>
+  //             </PrimaryButton>
+  //           </StyledAddCancelButtonContainer>
+  //         </StyledFarmInfo>
+  //       </Form>
+  //     )}
+  //   </Formik>
+  // );
```
Comment (GDamaso) [inline frontend/src/Commons/Forms/InputModules/Fertilizers/Fertilizers.tsx:314]: :eyes:

File: frontend/src/Commons/Forms/InputModules/ListComponent.styles.ts [hunk 2]
```diff
@@ -125,6 +133,14 @@ const StyledCropsGroup = styled.div`
   }
 `;
 
+const StyledCustomFertilizerGroup = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
```
Comment (GDamaso) [inline frontend/src/Commons/Forms/InputModules/ListComponent.styles.ts:139]: Didn't know about `space-evenly` nice :+1:

**Review by GDamaso — COMMENTED** [review]
Good job man, that was quick :rocket: 

Could we make the % in line with the Nutrient so it doesn't wrap? It moves their values one line down:

image

And how about adding the StyledDivider between the list items?

Good job man :fire:

File: frontend/src/Commons/Forms/InputModules/Fertilizers/Fertilizers.tsx (hunk 1 above)
Comment (kcaparas1630) [reply frontend/src/Commons/Forms/InputModules/Fertilizers/Fertilizers.tsx:314]: lmao I forgot

**Comment by kcaparas1630** [comment]
@GDamaso  I'm not sure in your part so I can't test it out.
here's how it look in mine. but just in case I added nowrap
image

**Review by GDamaso — APPROVED** [review]
:rocket:

**Pull request merged**